### PR TITLE
Echarts -> Improved Line Graph and merged with current master (other PR)

### PR DESCRIPTION
**Pull request by Steffeng5**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -104,3 +104,5 @@ dist
 .tern-port
 .vscode/arduino.json
 .vscode/c_cpp_properties.json
+
+.idea/
```
Comment (opatut) [inline .gitignore:108]: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Please do that instead.

**Review by opatut — COMMENTED** [review]
You can work on `openbikesensor:echarts` directly if you want, no need for PRs into that branch ;)

File: src/app/track/track.component.ts [hunk 2]
```diff
@@ -90,44 +56,117 @@ export class TrackComponent implements OnInit {
...
+            }))
+        const right = this.trackData.filteredPoints.filter(p => p.d2).map(p => [DateTime.fromFormat(`${p.date} ${p.time}`, 'dd.MM.yyyy HH:mm:ss').toMillis(), p.d2])
+        const rightTagged = this.trackData.filteredPoints.filter(p => p.d2 && p.flag)
+            .map(p => ({
+                xAxis: DateTime.fromFormat(`${p.date} ${p.time}`, 'dd.MM.yyyy HH:mm:ss').toMillis(),
+                yAxis: p.d2,
+                value: p.d2
+            }))
+
+        console.log("TS: " + timestamps.length)
```
Comment (groma84) [inline src/app/track/track.component.ts:103]: Please remove logs before merge

**Comment by groma84** [comment]
Please rebase again after the other branch is merged, so the duplicate commits are gone from this one. Then we can review and merge this one.

**Review by groma84 — APPROVED** [review]

**Pull request merged**